### ISSUE TITLE
Support setting switches/variables from command line arguments and/or RPG_RT.ini

### ISSUE THREAD

**Issue by fdelapena** [EasyRPG/Player#2477]
This is a feature request allowing some games to tweak values for debugging reasons or set by an external launcher in a generic way not requiring to patch the executable to customize behavior from external parameters.

**Comment by Soeufans** [comment]
This could be great, like to do something like "First time load"

Like something like to activate/deactive a swotch or a value in a variable when you play first time a game, to help skip some unwated intro, first configs and more :)

**Comment by fmatthew5876** [comment]
Makes sense as a useful way to enable debug systems. HH3 has a built in debugging engine enabled by switches.
